packages/__examples__/.storybook/stories/stories.ts

```diff
@@ -84,6 +84,8 @@ const additionalExamples: AdditionalExample[] = [
   }
 ]
 
+// Note: require.context causes Webpack to have huge compile times (and often
+// fail with out of memory errors) when switching to `pnpm`
```

Comment:
I could not find one and I've spent a total of ~15 hours trying to fix this.

But we will need to totally rewrite this file anyway since storiesOf was removed by Storybook in v8